### PR TITLE
Jin

### PR DESCRIPTION
**Pull request by ghost**
Here.

### CODE REVIEW

**Review by blchu — CHANGES_REQUESTED** [review]
let's keep all the data files (.pickle, .npy, .ckpt) out of the github repo and keep them somewhere else e.g. google drive; i'd like the have the repo contain only scripts for now

File: src/story_parse.py [hunk 1]
```diff
@@ -292,6 +327,8 @@ def __init__(self, book_urls, text_or_url, num_words, max_length,
         self.num_unique_words = len(self.decode_dict)
         self.all_sentences = self._get_all_sentences()
         self.all_stories = self._get_all_stories()
+        if embedding_path:
+            self.embeddings = self._get_embeddings(embedding_path)
```
Comment (blchu) [inline src/story_parse.py:329]: is self.embeddings used anywhere?

Comment (ghost) [reply src/story_parse.py:329]: no, im using self.embeddings as the word vectors, as opposed to self.encode_dict, which contains one hot indices

**Comment by ghost** [comment]
testing slack integration. Also, please approve :)

**Pull request merged**